packages/office-ui-fabric-react/src/components/Button/CommandButton/CommandButton.scss

```diff
@@ -9,61 +9,61 @@
 
 $button-commandButtonHeight: 40px;
 
-.ms-Button--command {
-  @include text-align(left);
+.ms-Button {
+  &.ms-Button--command {
```

Comment:
Makes sense, I wasn't considering that we now have race conditions to deal with. 👍